### PR TITLE
Spelling and grammar fixes + copyright year bump in README

### PR DESCRIPTION
**Pull request by somtijds**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Just grammar/spelling things + a copyright year bump

* **What is the current behavior?** (You can also link to an open issue here)
Does not apply.

* **What is the new behavior (if this is a feature change)?**
Does not apply.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No.

* **Other information**:

### CODE REVIEW

**Comment by Chrico** [comment]
Nice! Maybe we should even remove the year...otherwise we need to change the year next year again. :)

**Comment by somtijds** [comment]
Removing it would work for me too, I generally like having that indicator of the 'age' of a package -- even when it's bumped, at least you know it's been touched without having to look at the commit history.
But to be honest I'm not certain what is required from a copyright statement from a legal perspective -- and what would best serve our purpose: it would be good to find out and decide, since our 'License guideline' snippet in Confluence still also includes a year in the copyright statement 🤷‍♂️

**Comment by bueltge** [comment]
I often write `since 2006 until today`
But there is not possible for a template, so we should remove them.

File: README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+## What are issue and PR templates?
+[Issue and pull request templates](https://blog.github.com/2016-02-17-issue-and-pull-request-templates/)
 
 ## How to create issue and PR templates?
 [About issue and pull request templates](https://help.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates)
 
-## License and Copyright
-Copyright (c) 2020 Inpsyde GmbH.
+## License and copyright
+Copyright (c) 2021 Inpsyde GmbH.
```
Comment (Chrico) [inline README.md]: we should never add the current year here. Because then we need to update every year the year to match the current year 😬

**Pull request merged**